R/cube-transforms.R

```diff
@@ -187,6 +198,86 @@ applyTransforms <- function(x,
 }
 
 
+makeInsertionFunctions <- function(var_cats, transforms, cats_in_array = NULL, ...) {
+    ### Insertions
+    # collate insertions with categories for rearranging and calculation purposes
+    # we need the categories to know what order the the cube cells should be in
+    # and where to insert insertions (as they are `anchor`ed) to a category id.
+    dots <- list(...)
+    if (!"include" %in% names(dots)) {
```

Comment:
Nitpicky but can we swap the conditionals here to avoid the `!`?  I.e. 
```
if ("include" %in% names(dots)){
    includes <- dots$include
}...
```
Alternately maybe some brackets around the `%in%` statement, I always  find negated `%in% statements a bit difficult to read.